client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -114,37 +122,27 @@ const UIComponent = (props: ControlProps) => {
   const rows = schemaOptions?.rows;
 
   const width = schemaOptions?.width ?? '100%';
-  const labelFlexBasis = `${FORM_LABEL_WIDTH}%`;
-  const inputFlexBasis = `${100 - FORM_LABEL_WIDTH}%`;
-  return (
-    <Box display="flex" alignItems="center" gap={1}>
-      <Box style={{ textAlign: 'end' }} flexBasis={labelFlexBasis}>
-        <FormLabel sx={{ fontWeight: 'bold' }}>{label}:</FormLabel>
-      </Box>
-      <Box
-        flexBasis={inputFlexBasis}
-        justifyContent={'flex-start'}
-        display="flex"
-      >
-        <DebouncedTextInput
-          data={data}
-          onChange={value => handleChange(path, value)}
-          inputProps={{
-            sx: { margin: 0.5, width },
-            disabled: !props.enabled,
 
-            FormHelperTextProps: error
-              ? { sx: { color: 'error.main' } }
-              : undefined,
-            required: props.required,
-            multiline,
-            rows,
-            error,
-            helperText,
-          }}
-        />
-      </Box>
-    </Box>
+  return (
+    <DetailInputWithLabelRow
+      label={label}
+      inputProps={{
+        value: text ?? '',
+        sx: { margin: 0.5, width },
+        onChange: e => onChange(e.target.value || ''),
+        disabled: !props.enabled,
+        error,
+        helperText,
+        FormHelperTextProps: error
+          ? { sx: { color: 'error.main' } }
+          : undefined,
+        required: props.required,
+        multiline,
+        rows,
+      }}
+      labelWidthPercentage={FORM_LABEL_WIDTH}
+      inputAlignment={'start'}
+    />
```

Comment:
Restored to what it was previously, now we're using the debounce hook rather than presentational component.